app/assets/stylesheets/pageflow/extendables.scss

```diff
@@ -0,0 +1,26 @@
+%pageflow-widget {
+  $widget-text-color: #fff !default;
+  $widget-standard-font: $standard-font !default;
+
+  font-family: $standard-font;
+  font-size: 1em;
+  color: $widget-text-color;
+}
+
+%pageflow-page-type-thumbnail{
```

Comment:
Selector `pageflow-page-type-thumbnail` should be written in lowercase with underscores<br>Opening curly brace `{` should be preceded by one space